<?xml version="1.0"?>
<launch>
    <arg name="real_world" default="false" />
    <arg name="no_start_pose" default="false" />

    <include file="$(find martian_mines)/launch/core.launch">
        <arg name="real_world" value="$(arg real_world)" />
        <arg name="no_start_pose" value="$(arg no_start_pose)" />
    </include>

    <group ns="uav0">
        <rosparam command="load" file="$(find martian_mines)/config/aruco.yaml"/>
        <node type="precision_landing.py" name="precision_landing" pkg="martian_mines" output="screen"/>
        <node type="detection.py" name="detection" pkg="martian_mines" output="screen"/>
        <node type="bbox_publisher.py" name="land_on_target" pkg="martian_mines" output="screen"/>
    </group>
</launch>